.bazelrc

```diff
@@ -1,81 +1,123 @@
+################################################################
+# Config from TensorFlow's .bazelrc.                           #
+#                                                              #
+# This is copied directly, with some irrelevant parts removed. #
+################################################################
```

Comment:
I've re-arranged the bazelrc to make it more maintainable in the future. There are now two sections: the first (the bulk of it) is copied directly from TF's bazelrc, but with some sections (e.g. relating to GPUs) removed. The much smaller second section at the bottom adds LCE-specific config.

Reply:
Do you think it would make sense to move these configs into a different `.tensorflow.bazelrc` file and include it at the top with `try-import %workspace%/.tensorflow.bazelrc` here?

Reply:
Yeah that's a very good idea actually :+1: